web/policies/organization_membership_policy.ex

```diff
@@ -52,7 +52,7 @@ defmodule CodeCorps.OrganizationMembershipPolicy do
     Organization
     |> Repo.get(membership.organization_id)
   end
-  defp fetch_membership(user, nil), do: nil
+  defp fetch_membership(_, nil), do: nil
```

Comment:
Would consider doing `_user` instead, just for clarity's sake. Will have same benefit but explicit about what the variable would be.